.swift-version

```diff
@@ -0,0 +1 @@
+3.0
```

Comment:
I also found out that Xcode's automatic migration puts "SWIFT_VERSION = 3.0" into the xcodeproj file (even though it was Swift 3.1 version doing the migration), so given that also the popular Swift projects updated to 3.1 are also stating "3.0" in their `.swift-version`, I am inclined to rather leave .swift-version at "3.0".

So I left it at "3.0" to be safe.